redis/commands/core.py

```diff
@@ -1501,16 +1501,28 @@ def expire(self, name: KeyT, time: ExpiryT) -> ResponseT:
             time = int(time.total_seconds())
         return self.execute_command("EXPIRE", name, time)
 
-    def expireat(self, name: KeyT, when: AbsExpiryT) -> ResponseT:
+    def expireat(self, name: KeyT, when: AbsExpiryT, option: str = None) -> ResponseT:
```

Comment:
@dogukanteber WDYT about making these options (NX, XX, etc) variables in the function header. IMHO it would watch our other functions.